int32 allied_type
---
float32 white_balance_auto_tolerance
int32 error